Exiled.CustomRoles/API/Features/CustomRole.cs

```diff
@@ -68,6 +68,11 @@ public abstract class CustomRole
         /// </summary>
         public abstract string Description { get; set; }
 
+        /// <summary>
+        /// Gets or sets the chance of this role.
+        /// </summary>
+        public abstract float Chance { get; set; }
```

Comment:
1. Breaking change, this should not be abstract.
2. Why would we include a spawn chance property in the first place, when we don't handle the spawning of custom roles ourselves?